imports/include/nisync.h

```diff
@@ -0,0 +1,1046 @@
+/****************************************************************************
+ *                                NI-Sync
+ *---------------------------------------------------------------------------
+ *   Copyright (c) National Instruments 2003-2018.  All Rights Reserved.
+ *---------------------------------------------------------------------------
+ *
+ * Title:    niSync.h
+ * Purpose:  NI-Sync
+ *           Instrument Driver Declarations.
+ *
+ ****************************************************************************/
+
+#ifndef __NISYNC_HEADER
+#define __NISYNC_HEADER
+
+#ifndef NIVISA_PXI
+#define NIVISA_PXI
+#endif
+
+#include <visa.h>
+
+#if defined(__cplusplus) || defined(__cplusplus__)
+extern "C" {
+#endif
+
+/****************************************************************************
+ *----------------- Instrument Driver Revision Information -----------------*
+ ****************************************************************************/
+#define NISYNC_MAJOR_VERSION                 __NISYNC_MAJOR_VERSION__ /* Inst. driver major version */
+#define NISYNC_MINOR_VERSION                 __NISYNC_MINOR_VERSION__ /* Inst. driver minor version */
```

Comment:
`__NISYNC_MAJOR_VERSION__`  and `__NISYNC_MINOR_VERSION__` are not defined. Let's set the version as follows:

```suggestion
#define NISYNC_MAJOR_VERSION                 21 /* Inst. driver major version */
#define NISYNC_MINOR_VERSION                 0 /* Inst. driver minor version */
```